src/internal/testing/TestScheduler.ts

```diff
@@ -221,7 +221,7 @@ export class TestScheduler extends VirtualTimeScheduler {
   }
 
   /** @nocollapse */
-  static parseMarblesAsSubscriptions(marbles: string | null, runMode = false): SubscriptionLog {
+  static parseMarblesAsSubscriptions(marbles: string | null, runMode = true): SubscriptionLog {
```

Comment:
Unfortunately, I don't think we should change this default, as the method is part of the `TestScheduler`'s public API. It's not marked as internal, so consumers will have it in their TypeScript definition and could be calling it. It should be marked as internal, but it isn't. Same goes for the `parseMarbles` method.

Reply:
Yeah, other libs depend on this, for example: https://github.com/timdp/swirly/blob/de269cbb93c2f82f98115347cb1ec2e42db3001d/packages/swirly-parser-rxjs/src/index.ts#L4

Reply:
I did not think of that. I changed the defaults back to how they where and am setting the `runMode` parameter from within the tests itself.

Reply:
Good catch, @cartant, I missed that entirely.